tests/testthat/test_library.R

```diff
@@ -87,5 +91,13 @@ context("Khiva Version tests")
 
 test_that("Test Version", {
   out <- Version()
-  expect_equal(out$result, "0.1.0")
+  
+  tags <- GET(url = 'https://api.github.com/repos/shapelets/khiva/tags')
```

Comment:
Same comment as in the other repos, I am fine with this. But we should be aware of the problems it might cause